scripts/integrate_locale_files.js

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
__nit:__ let's keep the same style we used for `i18n_check` and name this file something like `i18n_integrate`?